include/SFML/Graphics/Color.hpp

```diff
@@ -37,7 +37,7 @@ namespace sf
 /// \brief Utility class for manipulating RGBA colors
 ///
 ////////////////////////////////////////////////////////////
-class SFML_GRAPHICS_API Color
+class Color
```

Comment:
With `constexpr`, the entire class is inline and header-only, so will not be linked, right?
(Except maybe the `static` members, depending on how they're declared).

Reply:
Yes, that is my understanding. CI seems to agree...